ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -213,6 +277,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-shared-vpc-phz-freeipa-f28
```

Comment:
If I remember right, the e2e job was added by auth team, the install workflow is using `cucushift-installer-rehearse-aws-ipi-shared-vpc-phz`, it is the same coverage as `aws-ipi-shared-vpc-phz-f14`, I do not think they need to be covered,  cc @xingxingxia to confirm.

Reply:
@jianlinliu , given I said `would like to cover openldap idp in upgrade` in another comment which will cover one LDAP idp in upgrade, we need not in upgrade to cover freeipa which is also LDAP idp but just is from different ldap implementation vendor. We can remove `aws-ipi-shared-vpc-phz-freeipa-f28` here.

Reply:
ok, removed aws-ipi-shared-vpc-phz-freeipa-f28